packages/auth-types/index.d.ts

```diff
@@ -325,6 +353,7 @@ export class FirebaseAuth {
   signInWithPopup(provider: AuthProvider): Promise<UserCredential>;
   signInWithRedirect(provider: AuthProvider): Promise<void>;
   signOut(): Promise<void>;
+  tenantId: string | null;
```

Comment:
Should we mark readonly properties? It is not clear when tenantId can be set and when it can only be read.

Reply:
done